jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/component/JwalaUi.java

```diff
@@ -181,8 +181,8 @@ public void click(final By by) {
         webElement.click();
     }
 
-    public void doubleClick() {
-        actions.doubleClick().perform();
+    public void backSpace() {
```

Comment:
Why does this work? Double click and back space are totally different actions

Reply:
Actually, we do delete "{" tag by selecting it with double click and send-keys over it. With new capabilities provided it was not functioning well. So, i removed the double click click functionality with backspace function + send keys which is exactly required for the operation